libbeat/processors/actions/decode_base64_field.go

```diff
@@ -46,34 +45,28 @@ type base64Config struct {
 	FailOnError   bool   `config:"fail_on_error"`
 }
 
-var (
-	defaultBase64Config = base64Config{
-		IgnoreMissing: false,
-		FailOnError:   true,
-	}
-)
-
 func init() {
 	processors.RegisterPlugin(processorName,
 		checks.ConfigChecked(NewDecodeBase64Field,
 			checks.RequireFields("field"),
-			checks.AllowedFields("field", "when")))
```

Comment:
Please don't delete `when`. It is required for conditional processing.

Reply:
fixed